packages/client/src/generation/TSClient/Datasources.ts

```diff
@@ -1,10 +1,10 @@
+import { DataSource } from '@prisma/generator-helper'
 import indent from 'indent-string'
 
-import type { InternalDatasource } from '../../runtime/utils/printDatasources'
 import type { Generatable } from './Generatable'
 
 export class Datasources implements Generatable {
-  constructor(protected readonly internalDatasources: InternalDatasource[]) {}
+  constructor(protected readonly internalDatasources: DataSource[]) {}
```

Comment:
Types were duplicated in the code, now we refer to a single source of truth. Cleanup was necessary.

Reply:
I remember them being a bit different 🤔